incubator/fluentd-cloudwatch/templates/daemonset.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "fullname" . }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+    kubernetes.io/cluster-service: "true"
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "fullname" . }}
+  template:
+    metadata:
+      labels:
+        app: {{ template "fullname" . }}
+        release: "{{ .Release.Name }}"
+        kubernetes.io/cluster-service: "true"
+      annotations:
+        {{- include "toYaml" .Values.annotations | indent 8 }}
```

Comment:
Helm now comes with a built-in `toYaml` function, so you can change this to:
```
      annotations:
{{- toYaml .Values.annotations | indent 8 }}
```

Note though, that it needs to be unindented.